### ISSUE TITLE
Button for outlook

### ISSUE THREAD

**Issue by ZAYEC77** [zurb/inky#84]
Hello. Thanks for inky.
What about possibility modify button tag like this gist:
https://gist.github.com/elidickinson/9424116#file-html_email_buttons_1-html

**Comment by rafibomb** [comment]
Yeah - totally want Bulletproof button in Foundation for Emails! Do you know if these work with % widths? Like our expanded button is 100% width.

**Comment by ZAYEC77** [comment]
Not sure about real outlook client, but I tested on litmus and it seems "v:roundrect" doesn't support width style in percent.
Maybe is possible to add some specific attribute for button tag to have ability add outlook hack with fixed width?
Something like: `<button outlook-width="300px"></button>`

What do you think about it?

**Comment by rafibomb** [comment]
Hmm, that could work, like if the width for outlook was set to the container size subtract padding.

**Comment by ZAYEC77** [comment]
I'm really weak in this thread
Is possible to integrate it into inky or it's like some separate case?

**Comment by joeworkman** [comment]
Duplicate of #325. Tracking bulletproof VML buttons there.
